src/files/BrsFile.ts

```diff
@@ -77,6 +77,27 @@ export class BrsFile {
         this.srcPath = value;
     }
 
+    /**
+     * Should this file be written to disk
+     */
+    public get isPublishable() {
+        let isPublishable = false;
+        this.ast.walk(createVisitor({
+            FunctionStatement: () => {
+                isPublishable = true;
+            },
+            MethodStatement: () => {
+                isPublishable = true;
+            },
+            ClassStatement: () => {
+                isPublishable = true;
+            }
+        }), {
+            walkMode: WalkMode.visitStatementsRecursive
```

Comment:
You probably don't need the recursive part (that steps into function bodies which we don't care about for this)

Reply:
Yeah, I wasn't sure if perhaps something like a namespace could contain functions so perhaps I needed the recursive version? Happy to change this and the methods version though. 👍

Reply:
Done!